src/Groups/perm.jl

```diff
@@ -639,6 +639,41 @@ function cycle_structures(G::PermGroup)
   return Set(cycle_structure(x) for x in r)
 end
 
+@doc raw"""
+    cycles(g::PermGroupElem)
+
+Return the cycles of the permutation `g` as a list of integer vectors.
```

Comment:
Does it return *all* cycles or only non-trivial ones? Maybe would be a good idea to pick an example with a trivial cycle (i.e. a fixed point) to show case that.

Reply:
It returns all cycles, I adjusted the example. Should one also write something about this in the doc string?